### DISCUSSION TITLE
Disable "notify" for errors

### DISCUSSION THREAD

**Discussion by yngwi** [folke/noice.nvim#108] [Q&A] [answered]
Hi,

I'm trying to use the "mini" view for all notifications and messages and have succeeded so far except for "Error" messages. Those are still opened in the notify-popup on the right bottom. How can I change those to "mini" as well?

Thanks!

**Comment by folke** [comment, marked as answer]
`config.messages.view_error`?

**Reply by yngwi** [reply]
Thank you
